docs/content/users/topics/phpstorm.md

```diff
@@ -2,51 +2,47 @@
 
 ## Full Integration with Docker, DDEV, and PhpStorm
 
-This explores how to add full PhpStorm integration with a DDEV project, including composer and phpunit. It works on all OS platforms, including macOS and WSL2.
-
-This is based on the wonderful [article](https://susi.dev/fully-integrate-ddev-and-phpstorm-including-unit-tests-with-coverage-update-2021) by [Susanne Moog](https://github.com/susannemoog), and couldn't have been done without it, but since PhpStorm has fixed some things and DDEV has worked around others, this is far easier now. It works on macOS, Linux, Windows with WSL2 and Windows traditional. You'll end up with PhpStorm actually using the PHP interpreter inside the ddev-webserver container of your project, and able to use composer and phpunit inside there as well.
+For full integration of PhpStorm and DDEV, it's easiest to use the [DDEV Integration Plugin](https://plugins.jetbrains.com/plugin/18813-ddev-integration) or in `Preferences -> Plugins -> Marketplace` search for `DDEV`. That does almost all of what is discussed here automatically, and works on all platforms.
 
 ### Requirements
 
-- PhpStorm 2021.3 or higher
-- You may need to add the `PHP Docker` plugin.
-- You need to enable Compose V2 Support in PHPStorm. This can be done in `Settings/Preferences | Build, Execution, Deployment | Docker | Tools` by either
-   1. set the bundled docker-compose binary as **Docker Compose executable** which can be found in `~/.ddev/bin`.
-   2. (Suggested for WSL) enabling **Use Compose V2** in PHPStorm and your Docker installation.
-- Any OS platform, but if you're using Windows PhpStorm with WSL2 the path mappings are slightly more complex. WSL2 instructions are provided where necessary.
+- PhpStorm 2022.2 or higher.
+- DDEV v1.20.1 or higher
 
 ### Setup Technique
 
-1. Open a DDEV project. In this example, the project name is "d9" and the site is "d9.ddev.site".
+1. Preferably, install the [DDEV Integration Plugin](https://plugins.jetbrains.com/plugin/18813-ddev-integration) so you don't have to do much of this.
```

Comment:
If you use the plugin you shouldn't need to do any of the following steps. So suggesting the plugin shoudln't be part of the list but an alternative to it.
This would also make the "(plugin will have done this)" redundant.

Reply:
Well, except the phpunit.

I do think this can be much better with a separate section for phpunit.